Sandboxie/install/Templates.ini

```
@@ -3771,6 +3771,16 @@ Tmpl.Title=Block DNS
 Tmpl.Class=Misc
 NetworkAccess=*,Block;Port=53;Protocol=UDP
 
+[Template_BlockAccessWMI]
+Tmpl.Title=Block Accessing WMI
+Tmpl.Class=Misc
+ClosedFilePath=%SystemRoot%\System32\wbem\*
+
+[Template_BlockLocalConnect]
+Tmpl.Title=Block Local Connect
+Tmpl.Class=Misc
+NetworkAccess=*,Block;Port=*;Address=*;Protocol=Any
```

This setting will block all connections. If this is the intention, you can remove the part after Block.

```suggestion
NetworkAccess=*,Block
```